pypst/utils.py

```diff
@@ -294,3 +295,33 @@ class Function(_RenderDataclass):
     """
 
     __is_function__: bool | str = True
+
+
+@dataclass
+class String:
+    """
+    String helper that wraps any content in quotes.
+
+    Example:
+        >>> line = (
+        ...     "The most 'common' string in programming is: "
+        ...     '"Hello world!".'
+        ... )
+        >>> s = String(line)
+        >>> print(s.render())
+        #"The most 'common' string in programming is: \\"Hello world!\\"."
+    """
+
+    body: str | Renderable | None = None
+
+    def __post_init__(self) -> None:
+        if isinstance(self.body, String):
+            self.body = self.body.body
+
+    def render(self) -> str:
+        """
+        Render the internal body to a string, escaping any symbols in JSON fashion.
+        """
+        body = json.dumps("" if self.body is None else render_code(self.body))
```

Comment:
`json.dumps` also escapes unicode characters (like äüö). Is this intended? If not, adding `ensure_ascii=False` would help.